models/TradingAccount.py

```diff
@@ -127,17 +129,9 @@ def getBalance(self, currency=''):
 
         if self.app.getExchange() == 'binance':
             if self.mode == 'live':
-                resp = self.client.get_account()
-                if 'balances' in resp:
-                    df = pd.DataFrame(resp['balances'])
-                    df = df[(df['free'] != '0.00000000') & (df['free'] != '0.00')]
-                    df['free'] = df['free'].astype(float)
-                    df['locked'] = df['locked'].astype(float)
-                    df['balance'] = df['free'] - df['locked']
-                    df.columns = [ 'currency', 'available', 'hold', 'balance' ]
-                    df = df[[ 'currency', 'balance', 'hold', 'available' ]]
-                    df = df.reset_index(drop=True)
-
+                model = AuthAPI(self.app.getAPIKey(), self.app.getAPISecret())
```

Comment:
This is the main change. Previously the `Binance.getAccount()` was being used only for the unit tests, so the Idea is that we start using the model to perform these operations.

Reply:
Looks like a good update and I agree that smaller PR's are better and easier to review.